### DISCUSSION TITLE
AsusWRT says Adguard address "not a valid IP address"

### DISCUSSION THREAD

**Discussion by rfehr613** [AdguardTeam/AdGuardHome#5836] [Q&A]
I'm trying to configure the Adguard addon in home assistant for my entire network.  I have an Aimesh configured with 2 AX86Us. When I go to the DNS configuration settings in AsusWRT and enter 127.0.0.1, AsusWRT says it's not a valid IP address. I tried 127.000.000.001, but that gave the same error. Interesting AsusWRT already has several DNS alternatives listed, several of which utilize Adguard, however none use IP addresses of 127.x.x.x.  What am I doing wrong here?

**Comment by machineh3d** [comment]
> I'm trying to configure the Adguard addon in home assistant for my entire network.  I have an Aimesh configured with 2 AX86Us. When I go to the DNS configuration settings in AsusWRT and enter 127.0.0.1, AsusWRT says it's not a valid IP address. I tried 127.000.000.001, but that gave the same error. Interesting AsusWRT already has several DNS alternatives listed, several of which utilize Adguard, however none use IP addresses of 127.x.x.x.  What am I doing wrong here?

Did you ever solve this issue?

**Reply by rfehr613** [reply]
Nope. Gave up
